dclab/rtdc_dataset/check.py

```diff
@@ -391,6 +391,36 @@ def check_fl_samples_per_event(self, **kwargs):
                             cfg_key="samples per event"))
         return cues
 
+    def check_fl_max_positive(self, **kwargs):
+        """ Check if all fl?_max values are >0.1 """
+        cues = []
+        neg_feats = []
+        for fl in ['fl1_max', 'fl2_max', 'fl3_max']:
+            if fl in self.ds:
+                if min(self.ds[fl]) <= 0.1:
+                    neg_feats.append(fl)
+        if neg_feats:
+            cues.append(ICue(
+                msg="Negative value for feature {}".format(neg_feats),
```

Comment:
To make this visually more appealing, I would write
`"Negative value for feature(s): {}".format(", ".join(neg_feats))`